src/hooks/siteDashboardHooks/useGetLastUpdated.ts

```diff
@@ -0,0 +1,46 @@
+import { useQuery, UseQueryOptions, UseQueryResult } from "react-query"
+
+import { LAST_UPDATED_KEY } from "constants/queryKeys"
+
+import useRedirectHook from "hooks/useRedirectHook"
+
+import * as SiteDashboardService from "services/SiteDashboardService"
+
+import { isAxiosError } from "utils/axios"
+import { convertUtcToTimeDiff } from "utils/dateUtils"
+import { useErrorToast } from "utils/toasts"
+
+const getRelativeLastUpdated = (siteName: string) => {
+  return SiteDashboardService.getLastUpdated(siteName).then((res) => {
+    return convertUtcToTimeDiff(res.lastUpdated)
+  })
+}
+
+export const useGetLastUpdated = (
+  siteName: string,
+  queryOptions?: Omit<UseQueryOptions<string>, "queryFn" | "queryKey">
+): UseQueryResult<string> => {
+  const { setRedirectToNotFound } = useRedirectHook()
+  const errorToast = useErrorToast()
+
+  return useQuery<string>(
+    [LAST_UPDATED_KEY, siteName],
+    () => getRelativeLastUpdated(siteName),
+    {
+      ...queryOptions,
+      retry: false,
+      onError: (err) => {
+        console.log(err)
```

Comment:
was this intentional to log to console?

Reply:
I don't have a clear answer for this because there are some other hooks that do this, but I removed it in a495c470.